networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -54,30 +57,37 @@ def generic_bfs_edges(G, source, neighbors=None):
     >>> G = nx.path_graph(3)
     >>> print(list(nx.bfs_edges(G,0)))
     [(0, 1), (1, 2)]
+    >>> list(nx.bfs_edges(G, source=0, depth_limit=2))
```

Comment:
This tests depth_limit of 2 but the full answer has depth limit of 2. So maybe its better to show depth_limit of 1 in this case.